common/util.go

```diff
@@ -760,3 +773,18 @@ func StringSetEqual(a, b []string) bool {
 	}
 	return StringSetEqual(b, a) // Above we checked only that all A are in B; check all B in A
 }
+
+// HandleSignal handles the passed in signal and calls the appropriate callback
+func HandleSignal(sig os.Signal, hostPort string, endpoint string, timeout time.Duration, handleFunc HandleSignalFunc) {
```

Comment:
@venkat1109 yeah.. this is used to hook up with the appropriate deployment frameworks.. by default i don't trap any signals.. so that our normal testing can go on.. it is in the other diff with our internal deployment framework integration..